This patch is for festival for Debian GNU/Linux, it adjusts server
access list, corrects spelling, ensures compliance with Filesystem
Hierarchy Standard and sets default audio backend to ALSA

Index: b/lib/festival.scm
===================================================================
--- a/lib/festival.scm
+++ b/lib/festival.scm
@@ -433,7 +433,7 @@
 to set this for the particular server task.
 [see Server/client API]")
 
-(defvar server_access_list '(localhost)
+(defvar server_access_list '("127.0.0.1" "localhost\.localdomain" "localhost")
   "server_access_list
 If non-nil this is the exhaustive list of machines and domains
 from which clients may access the server.  This is a list of REGEXs
@@ -494,7 +494,7 @@
 (If compiled with editline)
   M-h               displays help on current symbol  
   M-s               speaks help on current symbol  
-  M-m               displays relevant manula page in local netscape
+  M-m               displays relevant manual page in local netscape
   TAB               Command, symbol and filename completion
   C-p or up-arrow   Previous command
   C-b or left-arrow Move back one character
@@ -552,13 +552,13 @@
 (define (intro)
 "(intro)
  Synthesize an introduction to the Festival Speech Synthesis System."
-  (tts (path-append libdir "../examples/intro.text") nil))
+  (tts (path-append datadir "/usr/share/doc/festival/examples/intro.text") nil))
 
 (define (intro-spanish)
 "(intro-spanish)
  Synthesize an introduction to the Festival Speech Synthesis System
  in spanish.  Spanish voice must already be selected for this."
-  (tts (path-append libdir "../examples/spintro.text") nil))
+  (tts (path-append datadir "/usr/share/doc/festival/examples/spintro.text") nil))
 
 (define (na_play FILENAME)
 "(play_wave FILENAME)
@@ -631,3 +631,7 @@ Compile all the scheme files in the library directory."
 (defvar mlsa_beta_param 0.0)
 
 (provide 'festival)
+
+; Make ALSA the default backend
+(Parameter.set 'Audio_Command "aplay -q -c 1 -t raw -f s16 -r $SR $FILE")
+(Parameter.set 'Audio_Method 'Audio_Command)
